2.4/test/run

```diff
@@ -342,10 +342,121 @@ noprealloc = true" > $config_file
     echo "  Success!"
 }
 
+function insert_and_wait_for_replication() {
+    local host
+    host=$1
+    local data
+    data=$2
```

Comment:
I wouldn't separate that assignments into separate lines when you can use 'local X=$Y', though.